app/test_main.py

```diff
@@ -1,3 +1,35 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected_result", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (-1, -1, [0, 0]),
+    (100, 100, [21, 17])
+])
+def test_get_human_age_with_param_deco(cat_age: int,
+                                       dog_age: int,
+                                       expected_result: list) -> None:
+    assert get_human_age(cat_age, dog_age) == expected_result, (
+        f"Function 'get_human_age' should return {expected_result}"
```

Comment:
You must write description like this for every param in parameterizer `id or ids` attribute above. Such as:
`id="expected human age 0 for cat age 0 and dog age 0"`